autotest/gcore/cog.py

```diff
@@ -389,7 +389,7 @@ def my_cbk(pct, _, arg):
     if sys.platform == 'darwin' and gdal.GetConfigOption('TRAVIS', None) is not None:
         assert got_cs != [0, 0, 0]
     else:
-        assert got_cs == [26293, 23439, 14955]
+        assert got_cs == [26293, 23439, 14955] or got_cs == [26228, 22085, 12992]
```

Comment:
@abellgithub @rouault the project could use pytest's parameterization feature https://docs.pytest.org/en/stable/parametrize.html so that it can have separated, isolated, platform-dependent tests instead of using if statements within one test.

Reply:
can you remove the "if sys.platform == 'darwin' and gdal.GetConfigOption('TRAVIS', None) is not None:" line. I'd expect the updated assertion will fix it

Reply:
Sure. Didn't want to be presumptuous.